sfcli/SmartFace.Cli/Commands/SubWatchlistMember/RegisterWatchlistMemberCmd.cs

```diff
@@ -25,20 +26,20 @@ public class RegisterWatchlistMemberCmd
         [Option("-p|--photos <FILE>", "", CommandOptionType.MultipleValue)]
         public string[] Photos { get; set; }
 
-        public RegisterWatchlistMemberCmd(IWatchlistMemberRegistrationManager manager)
+        public RegisterWatchlistMemberCmd(IWatchlistMemberRegistrationManager registrationManager)
         {
-            Manager = manager;
+            _registrationManager = registrationManager;
         }
 
-        protected virtual int OnExecute(CommandLineApplication app, IConsole console)
+        protected virtual async Task<int> OnExecuteAsync(CommandLineApplication app, IConsole console)
         {
             var data = new RegisterWatchlistMemberExtended
             {
                 ExternalId = ExternalId,
                 PhotoFiles = Photos,
                 WatchlistExternalIds = WatchlistExternalIds
             };
-            Manager.RegisterWatchlistMember(data);
+            await _registrationManager.RegisterWatchlistMemberAsync(data);
```

Comment:
this command could print out json response now, before register did not returned anything.

Reply:
Added the logic of returning data back to API. Please check if the TPL dataflow-related code is OK